Services/DynamicEventHandler.cs

```diff
@@ -1,6 +1,6 @@
 ﻿/*
  * Events are more than just a convenient way of calling into all the registered dependencies. Actually calls to event 
- * go through the so-called Orchard Event Bus (see: http://docs.orchardproject.net/Documentation/How-Orchard-works#EventBus). 
+ * go through the so-called Orchard Event Bus (see: http://docs.orchardcore.net/projects/O1/en/latest/Documentation/How-Orchard-works#EventBus). 

```

Comment:
The correct URL here is: https://docs.orchardcore.net/projects/O1/en/latest/Documentation/How-Orchard-works/#event-bus